schema/artist/index.js

```diff
@@ -111,6 +111,20 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      formatted_nationality_and_birthday: {
+        type: GraphQLString,
+        description: 'A string of the form "Nationality, Birthday"',
+        resolve: ({ birthday, nationality }) => {
+          let formatted_bday = (!isNaN(birthday) && birthday) ? 'b. ' + birthday : birthday;
+          formatted_bday = formatted_bday ? formatted_bday.replace(/born/i, 'b.') : null;
```

Comment:
Yeah that makes sense, you can’t call functions on null.

Nit -pick for the future, I think I prefer the `&&` style over a ternary and returning `null`:

```js
formatted_bday = formatted_bday && formatted_bday.replace(/born/i, 'b.');
```